src/components/layout/layout.tsx

```diff
@@ -127,6 +129,8 @@ const Layout = (props: LayoutProps): JSX.Element => {
         siteTitleUrl={props.siteTitleUrl}
         $lightTheme={isLight}
         transparent={props.transparentHeader || Boolean(props.heroImage)}
+        showLanguageSwitch={props.showLanguageSwitch}
```

Comment:
Do we even need this? Or should we just always show translations if they are available? And if we don't have translations, we don't show the button